index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porofolio main page</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/091ffb7972.js" crossorigin="anonymous"></script>
+</head>
+<body>
+      <header class="head">
+        <nav>
+          <h2 class="pagename" >Welcome</h2>
+        </nav>
+        <img src="tools/Union.svg" alt="" class="nav-icon">
+    </header>
+    <section class="welcome">
+        <p ><span class="greetings">Hey there. I’m Josue</span><br>
+        <span class="profession">I’m a software developer</span></p>
+        <p class="description">I'm a student who is in his first week of the microverse module,
+             this is my first project I'm working on to see my advencements or progress.</p>
+           <footer class="socials">
+            <img src="tools/twitter.svg" alt="">
+            <img src="tools/Linkedin icon.svg" alt="">
+            <img src="tools/medium.svg" alt="">
+            <img src="tools/github.svg" alt="">
+            <img src="tools/angellist.svg" alt="">
+           </footer>
+           <img src="tools/Disabled.svg" alt="">
+    </section>
+  </body>
```

Comment:
![extra space](https://user-images.githubusercontent.com/93039359/198336995-2d6c51c2-1670-43a8-b381-2734acaf2751.PNG)

- [ ] You can see there is an extra space at the bottom of your design. Kindly remove that to make your design look similar to the Figma template 👍